src/modules/welcome/WelcomePage.cpp

```diff
@@ -77,11 +77,25 @@ WelcomePage::WelcomePage( Config* conf, QWidget* parent )
     ui->aboutButton->setIcon( CalamaresUtils::defaultPixmap(
         CalamaresUtils::Information,
         CalamaresUtils::Original,
-        2 * QSize( CalamaresUtils::defaultFontHeight(), CalamaresUtils::defaultFontHeight() ) ) );
+        2 * QSize( defaultFontHeight, defaultFontHeight ) ) );
     connect( ui->aboutButton, &QPushButton::clicked, this, &WelcomePage::showAboutBox );
 
+    // insert system-check widget below welcome text
     int welcome_text_idx = ui->verticalLayout->indexOf( ui->mainText );
     ui->verticalLayout->insertWidget( welcome_text_idx + 1, m_checkingWidget );
+
+    // insert optional logo banner image above welcome text
+    Calamares::Branding::ImageEntry bannerImage = Calamares::Branding::ProductBanner;
```

Comment:
I don't see the point here: you only use this enum value once, and the line doesn't get too long.

Reply:
i dont understand the dispute - the LOC referencing 'bannerImage' is nearly 90 chars now, even with the variable substituted